### ISSUE TITLE
NPM package is not actually upgraded

### ISSUE THREAD

**Issue by tituspijean** [YunoHost-Apps/nocodb_ynh#22]
### Describe the bug

Due to the constraints in `package.json`, NocoDB will not be actively upgraded with a simple `npm install`:

```
  "dependencies": {
    "dotenv": "^8.2.0",
    "express": "^4.17.1",
    "nocodb": "^0.84.8"
  }
```

Upstream advises to `npm uninstall nocodb` then `npm install --save nocodb`. However we need to fix the version number.
As per https://github.com/YunoHost-Apps/nocodb_ynh/issues/13, we also cannot trust the `^` version controller. Let's try to fix these versions in `package-lock.json`

**Comment by kpalik** [comment]
any progress on that? I installed 0.99.2 version while ago but after that i saw that despite upgrade to 0.100.x in this repository my app is showing 0.99.x version, even after uprading in yunohost admin. 

Then, i've uninstalled app and installed it fresh but it shows 0.99.2 version (installed today)

![obraz](https://user-images.githubusercontent.com/2924034/212676238-d01a8659-bd4b-4168-9413-6b8eb7a25b6c.png)

**Comment by tituspijean** [comment]
I am facing an issue when upgrading from v0.98.4 where an internal NocoDB database migration fails:

```
migration file "nc_021_add_fields_in_token" failed
migration failed with error: alter table `nc_api_tokens` add constraint `nc_api_tokens_fk_user_id_foreign` foreign key (`fk_user_id`) references `nc_users_v2` (`id`) - Can't create table `nocodb`.`nc_api_tokens` (errno: 150 "Foreign key constraint is incorrectly formed")
Error: alter table `nc_api_tokens` add constraint `nc_api_tokens_fk_user_id_foreign` foreign key (`fk_user_id`) references `nc_users_v2` (`id`) - Can't create table `nocodb`.`nc_api_tokens` (errno: 150 "Foreign key constraint is incorrectly formed")
    at Packet.asError (/opt/yunohost/nocodb/node_modules/mysql2/lib/packets/packet.js:728:17)
    at Query.execute (/opt/yunohost/nocodb/node_modules/mysql2/lib/commands/command.js:29:26)
    at Connection.handlePacket (/opt/yunohost/nocodb/node_modules/mysql2/lib/connection.js:456:32)
    at PacketParser.onPacket (/opt/yunohost/nocodb/node_modules/mysql2/lib/connection.js:85:12)
    at PacketParser.executeStart (/opt/yunohost/nocodb/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.<anonymous> (/opt/yunohost/nocodb/node_modules/mysql2/lib/connection.js:92:25)
    at Socket.emit (node:events:513:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9) {
  code: 'ER_CANT_CREATE_TABLE',
  errno: 1005,
  sqlState: 'HY000',
  sqlMessage: 'Can\'t create table `nocodb`.`nc_api_tokens` (errno: 150 "Foreign key constraint is incorrectly formed")',
  sql: 'alter table `nc_api_tokens` add constraint `nc_api_tokens_fk_user_id_foreign` foreign key (`fk_user_id`) references `nc_users_v2` (`id`)'
}
```

**Comment by KvL159** [comment]
I was able to run the latest version, I made a pull request for testing:
https://github.com/YunoHost-Apps/nocodb_ynh/pull/78

It also fixes the attachment issue (however I couldn't fix the 1mb upload limit yet)

**Comment by tituspijean** [comment]
Fixed by #79.
